Logger/BaseLoggerMixins.cs

```diff
@@ -1,7 +1,71 @@
 ﻿namespace Logger
 {
+    using System;
+
     public static class BaseLoggerMixins
     {
-        
+        public static void Error(this BaseLogger baseLogger, string message, params int[] extraParams)
```

Comment:
```suggestion
        public static void Error(this BaseLogger baseLogger, string message, params object[] extraParams)
```
You may consider passing an object array here instead.